Resources/Prototypes/Recipes/Crafting/Graphs/cratesecgear.yml

```diff
@@ -0,0 +1,30 @@
+- type: constructionGraph
+  id: CrateSecgear  
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: cratesecgear
+          steps:
+            - material: Steel
+              amount: 5
+              doAfter: 5
+
+              
+    - node: cratesecgear
+      entity: CrateSecgear
+      edges:
+        - to: start
+          steps:
+            - tool: Screwing
+              doAfter: 5
+          conditions:
+            - !type:StorageWelded
+              welded: false
+            - !type:Locked
+              locked: false              
+          completed:
+            - !type:SpawnPrototype
+              prototype: SheetSteel1
+              amount: 5
+            - !type:DeleteEntity {}
```

Comment:
theres still a fuck ton of duplication across all these, i get why its done but at the very least put all of these in their own folder

Reply:
Wasn't sure if sticking all the crates in their own construction folder would still function. Will do that if that's alright.

Again wasn't sure if having one graph that applied to the crate base entity would work across all crates but when I tested with lockers it didn't work.